### PR TITLE
i18n: Align field labels

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

File: pootle/apps/pootle_app/forms.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ class ProjectForm(forms.ModelForm):
     source_language = forms.ModelChoiceField(label=_('Source Language'),
                                              queryset=Language.objects.none())
     fs_plugin = forms.ChoiceField(
-        label=_('FS Plugin'),
+        label=_('Filesystem backend'),
```
Comment (phlax) [inline pootle/apps/pootle_app/forms.py]: this isnt actually displayed anywhere - not sure if we can just remove it

Comment (unho) [reply pootle/apps/pootle_app/forms.py]: I think you are right. I will adjust the PR.

File: pootle/apps/pootle_app/forms.py [hunk 2]
```diff
@@ -52,11 +52,8 @@ def clean_specialchars(self):
 
 class ProjectForm(forms.ModelForm):
 
-    source_language = forms.ModelChoiceField(label=_('Source Language'),
```
Comment (phlax) [inline pootle/apps/pootle_app/forms.py]: this one should also probably be removed

**Comment by phlax** [comment]
lgtm

**Pull request merged**